composer.json

```diff
@@ -50,21 +50,19 @@
         "friendsofsymfony/rest-bundle": "^2.8 || ^3.0",
         "gedmo/doctrine-extensions": "^2.4 || ^3.0",
         "goodby/csv": "^1.3",
-        "guzzlehttp/guzzle": "^6.2",
-        "guzzlehttp/psr7": "^1.5",
+        "guzzlehttp/promises": "^1.4",
         "handcraftedinthealps/rest-routing-bundle": "^1.0",
         "imagine/imagine": "^0.6.1 || ^0.7.0 || ^1.0",
         "jms/serializer": "^3.0",
         "jms/serializer-bundle": "^3.0",
         "massive/build-bundle": "^0.3 || ^0.4 || ^0.5",
         "massive/search-bundle": "^2.0",
+        "nyholm/psr7": "^1.3",
```

Comment:
This are the new recommended packages for using the FOSHttpCacheBundle with the Symfony http-client: https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/pull/561